tests/robustness/README.md

```diff
@@ -33,6 +33,30 @@ Upon failure tests generate a report that can be used to attribute whether failu
     * `RESULTS_DIR` - to change location where results report will be saved.
     * `PERSIST_RESULTS` - to persist the results report of the test. By default this will not be persisted in the case of a successful run.
 
+## Re-evaluate existing report
+
+Robustness test validation is constantly changing and improving.
+Errors in etcd model could be causing false positives, which makes the ability to re-evaluate the reports after we fix the issue important.
+
+> Note: Robustness test report format is not stable, and it's expected that not all old reports can be re-evaluated using the newest version.
+
+1. Identify location of the robustness report.
+
+   For local runs this would be by identifying log line, in the following example that would be `/tmp/TestRobustnessExploratory_Etcd_HighTraffic_ClusterOfSize1`:
+   ```
+   logger.go:146: 2024-04-08T09:45:27.734+0200 INFO    Saving robustness test report   {"path": "/tmp/TestRobustnessExploratory_Etcd_HighTraffic_ClusterOfSize1"}
+   ```
+
+   For remote runs on CI you need to go to the Github Actions page, download one of the Artifacts, extract it locally and pick one of the directories located within:
+
+   ![github actions artifact](./github_actions_artifacts.png)
```

Comment:
Considering we will only have reports on a failed run, can we add a screenshot of an Action with atleast one failed workflow?
Ex: https://github.com/etcd-io/etcd/actions/runs/8580223186 

Based on this, let's highlight just `release-34` to be downloaded.

Reply:
SG